docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-status.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: mesheryctl system status | Meshery
+permalink: reference/mesheryctl/system/status
+type: reference
+display-title: "false"
+language: en
+lang: en
+categories: en
+list: exclude
```

Comment:
What does “list: exclude” control, @alphaX86?

Reply:
Nothing much @leecalcote it's copied directly from template page.... It didn't return error when removed, so we can exclude safely.

Reply:
This means we need to remove these on every file and check 👀👀